### ISSUE TITLE
"dc-rendered" entry preview does not respect input system font setting

### ISSUE THREAD

**Issue by megahirt** [sillsdev/web-languageforge#1012]
Input systems are allowed to specify a font-family attribute for displaying data in a font that the local user is presumed to already have installed.  For example:

![image](https://user-images.githubusercontent.com/3444521/126737633-2716a913-5a7b-4847-807f-1c2cd65e6b94.png)

The "font name" field is injected as a CSS font-family attribute for every place where the data of that input system is displayed.  At least that is the intention.  This can be hard to notice for most scripts.  Burmese script however will not render properly on the default system font provided by Windows 10 (thus we specify Padauk for this script).

Here is a screenshot of the successfully injected font-family into a wrapper element around the multitext input tag.

![image](https://user-images.githubusercontent.com/3444521/126738759-99b0a677-c7d9-4351-b309-f2f9b0a160c8.png)

It's unclear to me why the dev tools have the font name strike-through.  I tested the font and it is actually working in Edge and Chrome and Firefox.

The issue is that this font-family injection is not happening in the dc-rendered entry preview area.

![image](https://user-images.githubusercontent.com/3444521/126743268-61bf7fd7-10bf-4e11-9f2a-1a7adaad0219.png)

There is currently no injection because the content which is display using ng-bind-html is already synthesized from several fields, so we cannot know exactly which input systems are used at this point.  What is needed is to break apart the dc-rendered into a more specific input-system level rendering and not synthesize things in Javascript.

See [dc-rendered.component.html](https://github.com/sillsdev/web-languageforge/blob/master/src/angular-app/languageforge/lexicon/editor/field/dc-rendered.component.html) and [dc-rendered.component.ts](https://github.com/sillsdev/web-languageforge/blob/master/src/angular-app/languageforge/lexicon/editor/field/dc-rendered.component.ts) for the current implementation

**Comment by rmunn** [comment]
> It's unclear to me why the dev tools have the font name strike-through. I tested the font and it is actually working in Edge and Chrome and Firefox.

CSS rules are stricken through when they are being overridden by another rule "later" in the CSS priority hierarchy. The `font-family: Padauk` in the style of the parent div might have been overridden by a `font-family` rule applying to its child `input` element. I'd have to verify that to be sure.

**Comment by megahirt** [comment]
> CSS rules are stricken through when they are being overridden by another rule "later" in the CSS priority hierarchy.

Yes, that is my understand as well, however in this case the rule is clearly still applying, because when I uncheck "font-family: Padauk" even though it is strike-through, I can see the font change, so the rule is still having an effect.  Additionally, unchecking the rule makes it disappear in the dev tools entirely for me on Edge.  It feels to me either like it's a bug in the dev tools, or they are indicating in this way that this is a non-standard font for which there is no web font defined, but they defer to the OS to try and use that font (which it does if it is installed).
